### ISSUE TITLE
Bit-HDTV - Unable to add due to reCAPTCHA added to login

### ISSUE THREAD

**Issue by testarea1** [Jackett/Jackett#1437]
Bit-HDTV tracker https://www.bit-hdtv.com has added a reCAPTCHA to their login page. Jackett is unable to add this tracker due to the reCAPTCHA. Error received:
```
Configuration failed: You didin't complete the reCAPTCHA! Login again
```

**Comment by kaso17** [comment]
should be fixed with 57b71732377f2814d1b29a863d6ec04d03ceeab9

**Comment by detkin** [comment]
It looks like the fix displays the reCAPTCHA but that it has been setup to not allow localhost as a domain. 

I get the reCAPTCHA with the message `Localhost is not in the list of supported domains for this site key.`

**Comment by kaso17** [comment]
you've to access jackett via the IP (127.0.0.1) not localhost, It's a recaptcha limitation

**Comment by detkin** [comment]
Yeah, that did the trick, thanks

**Comment by testarea1** [comment]
Anyone getting errors again? Now I keep getting "String was not recognized as a valid DateTime" and it's no longer pulling results. Error log:

```
Error on 20170607191043 for BIT-HDTV.txtSystem.FormatException: String was not recognized as a valid DateTime.
  at System.DateTimeParse.ParseExact (System.String s, System.String format, System.Globalization.DateTimeFormatInfo dtfi, System.Globalization.DateTimeStyles style) [0x0002a] in <a07d6bf484a54da2861691df910339b1>:0 
  at System.DateTime.ParseExact (System.String s, System.String format, System.IFormatProvider provider) [0x00008] in <a07d6bf484a54da2861691df910339b1>:0 
  at Jackett.Indexers.BitHdtv+<PerformQuery>c__async2.MoveNext () [0x00359] in C:\projects\jackett\src\Jackett\Indexers\BitHdtv.cs:173 
```

**Comment by detkin** [comment]
Yeah, I'm having the same problem now

**Comment by kaso17** [comment]
they made more changes, fixed with 81f2c7e91c9d25eaf86c6ab75f29dc07315d8eea
